v3/lint/configuration_test.go

```diff
@@ -0,0 +1,913 @@
+package lint
```

Comment:
These tests require custom structs for each case that we would like to exercise. That, coupled with the fact the types are not values in Go, means that table driven testing cannot reasonably be done (at least, I don't immediate see a way - please speak up if you can think of one).